lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -41,15 +41,40 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     t2 = Task.async(fn -> get_transaction_content(address) end)
 
     with {:ok, {:ok, genesis_address}} <- Task.yield(t1),
-         {:ok, {:ok, transaction_content}} <- Task.yield(t2) do
+         {:ok,
+          {:ok,
+           %{
+             "name" => name,
+             "supply" => supply,
+             "symbol" => symbol,
+             "type" => type,
+             "properties" => properties
+           }}} <- Task.yield(t2) do
+      data_to_digest =
+        case type do
+          "fungible" ->
```

Comment:
I think we should do in the other way, add the properties in the hash of fungible tokens and not in the non-fungible token as non_fungible token will probably have different properties

Reply:
Oh yes you re right ! I inverted the logic 😅